src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java

```diff
@@ -501,9 +501,6 @@ private void addTag(String text, int line) {
                 format = CommonUtil.fillTemplateWithStringsByRegexp(
                         filter.influenceFormat, text, filter.commentFormat);
 
-                if (CommonUtil.startsWithChar(format, '+')) {
```

Comment:
This is the only change in src/main - I don't think it's necessary to strip a leading "+" here, because this string gets sent to `Integer.parseInt` below which can handle a leading "+" (I updated the `testInfluenceFormat` case to make that explicit in the test)